docs/new-feature-tutorial.md

```diff
@@ -77,6 +77,13 @@ module are located in `frontend_tests/node_tests/`. For more
 information on writing and running tests see the [testing
 documentation](testing.html).
 
+### Documentation changes
+
+After implementing the new feature, it is recommended that you
```

Comment:
"After implementing the new feature, you should document it" (we generally don't like passive tense/helping verbs in the docs)